qtism/data/storage/xml/XmlDocument.php

```diff
@@ -443,30 +443,43 @@ public function includeAssessmentSectionRefs($validate = false)
      */
     protected function decorateRootElement(DOMElement $rootElement)
     {
-        $qtiSuffix = 'v2p1';
-        $xsdLocation = 'http://www.imsglobal.org/xsd/qti/qtiv2p1/imsqti_v2p1.xsd';
         switch (trim($this->getVersion())) {
             case '2.0':
                 $qtiSuffix = 'v2p0';
                 $xsdLocation = 'http://www.imsglobal.org/xsd/imsqti_v2p0.xsd';
                 break;
 
+            case '2.2':
```

Comment:
What do u think about taking under account `2.0.0` and `2.2.0` (if exists of course)

Reply:
On this matter, all we have are 2.0, 2.1 and 2.2 (3.0 coming). I'll clean it up and ensure that the version is validated before creating the object.